doc/types/Progress.qbk

```diff
@@ -8,8 +8,8 @@
 [section:Progress Progress]
 
 A [*Progress] object provides feedback to callers on the progress of
-long running operations such as calls to [link nudb.ref.verify verify] or
-[link nudb.ref.rekey rekey] which can take days or weeks for database that
+long running operations such as calls to [link nudb.ref.nudb__verify verify] or
+[link nudb.ref.nudb__rekey rekey] which can take days or weeks for database that
```

Comment:
Last three words in tis line are "_for database that_": they make no grammatical sense. Use plural: _"for databases that"_ to fit with the existing sentence.